src/current/cockroachcloud/aws-privatelink.md

```diff
@@ -1,128 +1,307 @@
+For broader context, refer to [Network Authorization for {{ site.data.products.db }} Clusters](network-authorization.html).
 
-This page describes the steps to setting up an AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster from your AWS VPC Console.
+This page describes the steps to setting up an AWS PrivateLink connection for {{ site.data.products.dedicated }} and multi-region {{ site.data.products.serverless }} clusters from your AWS VPC Console.
 
-{{site.data.alerts.callout_success}}
+{{site.data.alerts.callout_info}}
+AWS PrivateLink for {{ site.data.products.serverless }} is in **[limited access]({{link_prefix}}cockroachdb-feature-availability.html)** and is only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. This feature is subject to change.
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="dedicated">{{ site.data.products.dedicated }}</button>
+  <button class="filter-button" data-scope="serverless">Multi-region {{ site.data.products.serverless }}</button>
+</div>
 
-You must also configure the AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster using the Console UI, Cloud API or Terraform Provider, so that {{ site.data.products.db }} can accept the new connection. For help, refer to [Establish VPC Peering or AWS PrivateLink](connect-to-your-cluster.html#establish-gcp-vpc-peering-or-aws-privatelink).
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+{{site.data.alerts.callout_success}}
+You must also configure the AWS PrivateLink connection for your {{ site.data.products.dedicated }} cluster using the {{ site.data.products.db }} Console, [Cloud API](cloud-api.html) or [Terraform Provider](provision-a-cluster-with-terraform.html),, so that {{ site.data.products.db }} can accept the new connection. For help, refer to [Establish VPC Peering or AWS PrivateLink](connect-to-your-cluster.html#establish-gcp-vpc-peering-or-aws-privatelink).
 
 If you have multiple clusters, you will have to repeat these steps for each cluster that you want to connect to using AWS PrivateLink.
 {{site.data.alerts.end}}
 
-### Set up a cluster
+</section>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+{{site.data.alerts.callout_success}}
+For each region where you deploy {{ site.data.products.serverless }} clusters with private connections, you must also configure the AWS PrivateLink connection for {{ site.data.products.serverless }} using the Console UI, Cloud API or Terraform Provider, so that {{ site.data.products.db }} can accept the new connection.
+
+All {{ site.data.products.serverless }} clusters in your organization that use private connections in a given region connect over the same shared VPC. You will have to complete these steps once per region you want to connect to using AWS PrivateLink.
```

Comment:
Why are we referring to shared VPC here? If you're referring to shared VPC on other side, then it's factually inaccurate because one could create private endpoints to different clusters in different VPCs. And if you're referring to shared VPC on our side, then that's unnecessary because serverless as a product already means it's shared network and other resources, and we don't need to belabor that here.